stable/hazelcast/templates/rbac.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.create -}}
```

Comment:
Please follow our best practices for RBAC:
https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

Reply:
Done.